src/leiningen/i18n/bin/update-pot.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -eo pipefail
+
+repo_dir="$(realpath "$(dirname "$0")/../../..")"
+pot_file="$repo_dir/locales/messages.pot"
```

Comment:
Tempted to delete this ^^ comment. Given may not be relevant.  More just hoping to elicit some thinking about how big of a thing that would be to change in future...